web/src/repogroups/HomepageConfig.ts

```diff
@@ -25,7 +25,7 @@ export const homepageLanguageList: { name: string; filterName: string }[] = [
     { name: 'Haskell', filterName: 'haskell' },
     { name: 'Html', filterName: 'html' },
```

Comment:
Shouldn't this be all-caps?
```suggestion
    { name: 'HTML', filterName: 'html' },
```